tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -1160,28 +1162,67 @@ export class WebGPUBackend extends KernelBackend {
 
   fromPixels(
       pixels: backend_util.PixelData|ImageData|HTMLImageElement|
-      HTMLCanvasElement|HTMLVideoElement,
+      HTMLCanvasElement|HTMLVideoElement|ImageBitmap,
       numChannels: number): Tensor3D {
     if (pixels == null) {
       throw new Error(
           'pixels passed to tf.browser.fromPixels() can not be null');
     }
 
+    // TODO(tfjs-optimization): These has chance to be cached and do not update
+    // buffer always.
     const outShape = [pixels.height, pixels.width, numChannels];
+    const size = util.sizeFromShape(outShape);
+    const uniformData = [size, pixels.width, numChannels];
     let imageData = (pixels as ImageData | backend_util.PixelData).data;
 
     if (env().getBool('IS_BROWSER')) {
       if (!(pixels instanceof HTMLVideoElement) &&
           !(pixels instanceof HTMLImageElement) &&
           !(pixels instanceof HTMLCanvasElement) &&
           !(pixels instanceof ImageData) &&
-          !(pixels.data instanceof Uint8Array)) {
+          !(!(pixels instanceof ImageBitmap) &&
```

Comment:
do we need to add another && condition to the pixels.data instanceof Uint8Array check?

Reply:
I think no need because it seems that if it is ImageData object, it should be Uint8Array https://developer.mozilla.org/en-US/docs/Web/API/ImageData

Reply:
Done.